src/components/Lesson/LessonDownloads/LessonDownloads.page.tsx

```diff
@@ -229,6 +230,9 @@ export function LessonDownloads(props: LessonDownloadsProps) {
 
   const { onSubmit } = useDownloadForm({ isLegacyDownload: isLegacyDownload });
 
+  const [isDownloadSuccessful, setIsDownloadSuccessful] =
+    useState<boolean>(false);
```

Comment:
now we have a few states this page could be in it might be better to combine them into one, eg `uiState = 'default' | 'downloading' | 'succeeded'` ?